inginious/frontend/pages/course_admin/settings.py

```diff
@@ -76,7 +76,7 @@ def POST_AUTH(self, courseid):  # pylint: disable=arguments-differ
                 errors.append(_('Invalid ACL value'))
             if course_content['registration_ac'] == "None":
                 course_content['registration_ac'] = None
-            course_content['registration_ac_list'] = data['registration_ac_list'].splitlines()
+            course_content['registration_ac_list'] = [re.sub(' ', '', line) for line in data['registration_ac_list'].splitlines()]
```

Comment:
``line.strip()`` is probably more appropriate.

Reply:
Do we accept entries such as `foo bar` in the ACL?

Reply:
I wasn't able to add tabs via the web interface but this is still possible via the yaml file.
I will replace the regex by `strip`.